bciers/apps/administration/app/data/jsonSchema/operationInformation.ts

```diff
@@ -170,12 +170,12 @@ const section3Schema: RJSFSchema = {
     },
   },
   dependencies: {
-    registration_category: {
+    registration_purpose: {
       allOf: [
         {
           if: {
```

Comment:
This schema was already set up with if/then and it appears to be working. It might be ~slightly better to use `oneOf` instead of `allOf` so we can do more with dynamic values (see the business structures in an operator's parent operators for an example). Thoughts?